conf/default.cfg

```diff
@@ -255,3 +255,7 @@ use = egg:swift3#swift3
 # OpenIO version
 use = egg:oioswift#copy
 object_post_as_copy = false
+
+[filter:verb_acl]
+use = egg:oioswift#verb_acl
+verb_acl = PUT:127.0.0.1,127.0.0.2;DELETE:127.0.0.3,127.0.0.4;PUT,DELETE:127.0.0.4,127.0.0.6
```

Comment:
Do you accept `192.168.1.0/24` ?

Reply:
Not yet.

Reply:
Is that a whitelist or a blacklist? Without reading the code it is not obvious.